examples/network_set.go

```diff
@@ -20,7 +20,7 @@ func main() {
 		os.Getenv("ONEVIEW_OV_DOMAIN"),
 		os.Getenv("ONEVIEW_OV_ENDPOINT"),
 		false,
-		2000,
```

Comment:
instead of changing the api version for every release, why don't to use apiversion variable
```
	apiversion, _ := strconv.Atoi(os.Getenv("ONEVIEW_APIVERSION"))
```

Reply:
done